api/proto/teleport/integration/v1/awsra_service.proto

```diff
@@ -0,0 +1,104 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.integration.v1;
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/integration/v1;integrationv1";
+
+// AWSRolesAnywhereService provides access to AWS APIs using the AWS Roles Anywhere Integration.
+service AWSRolesAnywhereService {
+  // AWSRolesAnywherePing does an health check for the integration.
+  // Returns the caller identity and the number of AWS Roles Anywhere Profiles that are active.
+  // It uses the following APIs:
+  // https://docs.aws.amazon.com/STS/latest/APIReference/API_GetCallerIdentity.html
+  // https://docs.aws.amazon.com/rolesanywhere/latest/APIReference/API_ListProfiles.html
+  rpc AWSRolesAnywherePing(AWSRolesAnywherePingRequest) returns (AWSRolesAnywherePingResponse);
+
+  // ListRolesAnywhereProfiles lists the AWS Roles Anywhere Profiles that are configured in the integration.
+  // It uses the following API:
+  // https://docs.aws.amazon.com/rolesanywhere/latest/APIReference/API_ListProfiles.html
+  // https://docs.aws.amazon.com/rolesanywhere/latest/APIReference/API_ListTagsForResource.html
+  rpc ListRolesAnywhereProfiles(ListRolesAnywhereProfilesRequest) returns (ListRolesAnywhereProfilesResponse);
+}
+
+// AWSRolesAnywherePingRequest is a request for doing an health check against the configured integration.
+message AWSRolesAnywherePingRequest {
+  // Integration is the AWS Roles Anywhere Integration name.
+  // Required if testing an existing integration, otherwise it should be empty and the other fields should be set.
+  string integration = 1;
+
+  // The AWS Roles Anywhere Trust Anchor ARN to be used when generating the token.
+  // Required if integration is empty.
+  string trust_anchor_arn = 2;
+
+  // The AWS Roles Anywhere Profile ARN to be used when generating the token.
+  // Required if integration is empty.
+  string profile_arn = 3;
+
+  // The AWS Role ARN to be used when generating the token.
+  // Required if integration is empty.
+  string role_arn = 4;
+}
+
+// AWSRolesAnywherePingResponse contains the response for the Ping operation.
+message AWSRolesAnywherePingResponse {
+  // The AWS account ID number of the account that owns or contains the calling entity.
+  string account_id = 1;
+  // The AWS ARN associated with the calling entity.
+  string arn = 2;
+  // The unique identifier of the calling entity.
+  string user_id = 3;
+  // The number of AWS Roles Anywhere Profiles that are active and have at least one associated Role.
+  int32 profile_count = 4;
+}
+
+// ListRolesAnywhereProfilesRequest is a request to list AWS Roles Anywhere Profiles.
+message ListRolesAnywhereProfilesRequest {
+  // Integration is the AWS Roles Anywhere Integration name.
+  string integration = 1;
+  // NextPageToken is used to paginate the results.
+  string next_page_token = 2;
+}
```

Comment:
nit: this should include a page size/limit parameter.  even if the current impl wants to always use some specific page size, its possible that future versions may want to have client-side control over page size and it would be preferable to have older servers understand the parameter if/when that happens.